package.json

```diff
@@ -24,7 +24,6 @@
   "peerDependencies": {
     "code-push": ">=1.8.0-beta",
     "cordova-plugin-file-transfer": ">=1.3.0",
-    "cordova-plugin-file": ">=3.0.0",
```

Comment:
The latest file transfer plugin depends on file `^4.0.0`. https://github.com/apache/cordova-plugin-file-transfer/blob/master/plugin.xml#L32 Do we want to upgrade to that? What was the version of the file plugin that fixed our Android permissions issue?